external/header.php

```diff
@@ -34,16 +34,17 @@ public static function __callstatic($name, $arguments)
 // Only users from authorized IP addresses may control Profiling
 if ($controlIPs === false || in_array($_SERVER['REMOTE_ADDR'], $controlIPs) || PHP_SAPI == 'cli')
 {
-  if (isset($_GET['_profile']))
+  if (isset($_GET[$_xhprof['getparam']]))
```

Comment:
I'm a bit biased here from library maintainer viewpoint I'd say the app has too general naming and that caused a collision. But I also understand your point and see no issue allowing changing parameter name as long as it won't break BC 😉